### PR TITLE
Add config-download for networker multinode adoption CI

### PR DESCRIPTION
**Pull request by marios**
As part of [1] adds the bits needed for the multinode OSP17 deployment with dedicated networker nodes.

[1] https://issues.redhat.com/browse/OSPRH-8810

### CODE REVIEW

File: devsetup/tripleo/overcloud_services.yaml [hunk 1]
```diff
@@ -40,7 +40,8 @@ parameter_defaults:
   DockerPuppetDebug: true
   ContainerCli: podman
   ControllerCount: 3
-  ComputeCount: 3
+  ComputeCount: 2
```
Comment (marios) [inline devsetup/tripleo/overcloud_services.yaml:43]: just for testing 
will find a better place to override this

File: devsetup/tripleo/tripleo_install.sh [hunk 2]
```diff
@@ -94,6 +109,6 @@ openstack overcloud deploy --stack overcloud \
     -e /home/zuul/containers-prepare-parameters.yaml -e /usr/share/openstack-tripleo-heat-templates/environments/podman.yaml \
     -e /usr/share/openstack-tripleo-heat-templates/environments/low-memory-usage.yaml \
     -e /usr/share/openstack-tripleo-heat-templates/environments/debug.yaml --validation-warnings-fatal ${ENV_ARGS} ${CEPH_OVERCLOUD_ARGS} \
-    -e /home/zuul/overcloud_services.yaml -e /home/zuul/config-download.yaml \
+    -e /home/zuul/overcloud_services.yaml -e /home/zuul/$config_download \
```
Comment (cescgina) [inline devsetup/tripleo/tripleo_install.sh:112]: if I understand correctly, now we have a `ComputeCount: 3` in `overcloud_services` and a `ComputeCount: 2` in the new `config-download-networker`. Do we know how the precendence is handled? Is it like ansible when only the last occurrence counts?

Comment (marios) [reply devsetup/tripleo/tripleo_install.sh:112]: yes exactly
with the tripleo-heat-templates environments last one wins and I am relying on that config-download-networker.yaml to come after services.yaml

it allows us to keep it simpler here with the networker vars in one file

Comment (cescgina) [reply devsetup/tripleo/tripleo_install.sh:112]: ack, in that case maybe I would left a comment noting that we rely on that order to get the right amount of computes, for future reference, other than that, the PR lgtm

**Comment by marios** [comment]
can someone add the lgtm label @raukadah or @viroel please if when you next have some time thank you!

**Pull request merged**